recipe/build.sh

```diff
@@ -75,6 +75,9 @@ if [[ $(uname) == "Linux" ]]; then
       SKIPS+=(-skip); SKIPS+=(qttools)
       SKIPS+=(-skip); SKIPS+=(qtlocation)
       SKIPS+=(-skip); SKIPS+=(qt3d)
+      EXTRA_OPTIONS=""
+    else
+      EXTRA_OPTIONS="-gstreamer 1.0"
```

Comment:
I don't think so. This option just avoided an error message when trying to build the "minimal" build. Since many things are skipped in the minimal build, it seemed that qt was error with an error message like "unkown option gstreamer". I've mostly "moved" this flag from just below.

Reply:
Just following up on this. I think this flag mostly says to use the 1.X series,  instead of the 0.10 series.

> GStreamer is released under the LGPL. The 1.x series is API and ABI stable and supersedes the previous stable 0.10 series. Both can be installed in parallel. 

https://gstreamer.freedesktop.org/